docs/devel/kubectl-conventions.md

```diff
@@ -279,15 +279,17 @@ type MineConfig struct {
   mineLatest bool
 }
 
-const (
-  mineLong = `Some long description
-for my command.`
+var (
+  mineLong = dedent.Dedent(`
+	Some long description
+	for my command.`)
 
-  mineExample = `  # Run my command's first action
-  $ %[1]s first
+  mineExample = dedent.Dedent(`
+	  # Run my command's first action
+	  $ %[1]s first
```

Comment:
Can you remove the `$` sign also?

Also `%[1]s first` looks mysterious to me. Can you change it to something like `kubectl <command> <first_action>`